screens/AppNavigator/screens/SettingsScreen/SettingsNavigator.tsx

```diff
@@ -17,8 +23,27 @@ export function SettingsNavigator (): JSX.Element {
       <SettingsStack.Screen
         name='settings'
         component={SettingsScreen}
-        options={{ headerTitle: translate('screens/SettingsScreen', 'Settings') }}
+        options={{
+          headerTitle: translate('screens/SettingsScreen', 'Settings'),
+          headerRight: () => <HelpButton />
```

Comment:
I think it's better to embed than have another screen. Since the settings are pretty empty now, no point separating it.

Reply:
I disagree with this. It follows the design for a reason. The entire help page is not a `Setting` that the user can edit. It would be bad UX to put it there. Also, in case we populate the settings page with some options in the future, we will be doing twice the work to relocate the help page again.

Reply:
I agree with @thedoublejay here. I see it as we are setting UX foundations for the app in a sense that we are distinguishing different goals from each screen – with one being for configuration and the other for information. In the future, we will be able to expand the help possibly with tutorial videos or other informative content which would benefit from having its own page.